$MirOS: ports/www/dillo/v2/patches/patch-dw_table_cc,v 1.1 2009/08/24 16:54:23 tg Exp $
--- dw/table.cc.orig	Mon Jan 18 18:00:19 2010
+++ dw/table.cc	Sun Feb 21 18:16:58 2010
@@ -23,6 +23,7 @@
 #include "../lout/msg.h"
 #include "../lout/misc.hh"
 
+#undef MAX
 #define MAX misc::max
 
 using namespace lout;
